docs/submit-opentargets-batch.md

```diff
@@ -5,57 +5,92 @@ Before starting the process, follow the [Build instructions](build.md). In parti
 
 Typical OpenTargets submission process consists of five major parts; for each one, an issue is created in the JIRA tracker. Issue template is linked for each of the steps, as well as the list of checks do be done during review.
 
+## Set up environment
+Commands below depend on a number of environment variables. It makes sense to set them all at once before executing any of the steps.
+
+```bash
+# Year and month for the upcoming OpenTargets release. This is announced in their e-mail.
+export OT_RELEASE=YYYY-MM
+
+# Year and month of ClinVar release used (see step 1.1 Download ClinVar data).
+# Note that this is *different* from the OpenTargets release year/month.
+export CLINVAR_RELEASE=YYYY_MM
+
+# This variable should point to the directory where this repository clone is located on the cluster.
+export CODE_ROOT=/nfs/production3/eva/software/eva-cttv-pipeline
+
+# Setting up Python version
+PYTHON_VERSION=3.5.6
+INSTALL_PATH=${HOME}/software/python-${PYTHON_VERSION}
```

Comment:
this points to your home, right? What do you think about putting it in a common eva/software/ folder? we have there an unrelated python 3.4.5 we use for other pipelines.

Reply:
Actually this points to `eva_etl`'s home (which I admittedly forgot to mention anywhere). Putting this Python into `eva/software/` is a good suggestion, will do. (Although I can't see python 3.4.5 there, only `python3-venv` folder which is not a complete installation but just a venv)